src/leap/mail/mail.py

```diff
@@ -744,7 +746,7 @@ def __cb_signal_unread_to_ui(self, unseen):
         :param unseen: number of unseen messages.
         :type unseen: int
         """
-        emit_async(catalog.MAIL_UNREAD_MESSAGES, str(unseen))
+        emit_async(catalog.MAIL_UNREAD_MESSAGES, self.store.uuid, str(unseen))
```

Comment:
I don't understand how mail_status handle the uuid, but I will trust you that this is fine and go ahead with it.